### ISSUE TITLE
mysterious devicemapper device appears and causes high IO

### ISSUE THREAD

**Issue by dmyerscough** [docker/docker#19521]
I am running Docker 1.9.0 on Debian Jessie, I noticed that a mysterious device appears for a few minutes that causes high IO on the box and then it clears. The high IO impacts the containers.

I cannot trace where the device comes from but you can see from the screenshot below `dm-12` util goes to 100%

![dm device](https://cloud.githubusercontent.com/assets/1395617/12465935/9fe35d84-bf85-11e5-9945-51d9c108f944.png)

I tried to do an `ls -al /dev/mapper/` to trace down the device and what container it might belong to but it doesnt show up.

```bash
Containers: 9
Images: 614
Server Version: 1.9.0
Storage Driver: devicemapper
 Pool Name: docker-252:0-39459481-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 214.7 GB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 15.46 GB
 Data Space Total: 214.7 GB
 Data Space Available: 199.3 GB
 Metadata Space Used: 37.91 MB
 Metadata Space Total: 4.295 GB
 Metadata Space Available: 4.257 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.90 (2014-09-01)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 3.18.19
Operating System: Debian GNU/Linux 8 (jessie)
CPUs: 24
Total Memory: 125.8 GiB
Name: appdocker72-sjc1
ID: IYPE:XGV6:P67Q:MQ7V:7NWX:5ROV:6KWT:R2HA:JG5C:6RKQ:K7QP:TECP
WARNING: No swap limit support
```

Has anyone else experienced this?

**Comment by dmyerscough** [comment]
I couldn't find anything similar to this issue.

BUG REPORT INFORMATION

Use the commands below to provide key information from your environment:

docker version: 1.9.0
docker info:
Containers: 9
Images: 614
Server Version: 1.9.0
Storage Driver: devicemapper
 Pool Name: docker-252:0-39459481-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 214.7 GB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 15.46 GB
 Data Space Total: 214.7 GB
 Data Space Available: 199.3 GB
 Metadata Space Used: 37.91 MB
 Metadata Space Total: 4.295 GB
 Metadata Space Available: 4.257 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.90 (2014-09-01)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 3.18.19
Operating System: Debian GNU/Linux 8 (jessie)
CPUs: 24
Total Memory: 125.8 GiB
Name: appdocker72-sjc1
ID: IYPE:XGV6:P67Q:MQ7V:7NWX:5ROV:6KWT:R2HA:JG5C:6RKQ:K7QP:TECP
WARNING: No swap limit support

Provide additional environment details (AWS, VirtualBox, physical, etc.): Physical

List the steps to reproduce the issue:
1. This is random and haven't found a way to reproduce
2.
3.

Describe the results you received:

High IO due to a unknown device mapper device appearing (Looks to be started by Docker)

Describe the results you expected:

Provide additional info you think is important:

**Comment by wb14123** [comment]
Hi, @dmyerscough . I've get the same problem with yours. Have you solve it?
